qt/python/mantidqt/widgets/instrumentview/presenter.py

```diff
@@ -10,6 +10,8 @@
 """
 Contains the presenter for displaying the InstrumentWidget
 """
+from PyQt5.QtCore import Qt
```

Comment:
This should be qtpy to be consistent:
```suggestion
from qtpy.QtCore import Qt
```